changelog/new_default_ddoc_theme.dd

```diff
@@ -0,0 +1 @@
+A completely new default Ddoc theme has been created.
```

Comment:
That's 3 adjectives in a row!
How about `The default Ddoc them has been overhauled.`?